src/module.c

```diff
@@ -8978,6 +8978,28 @@ void ModuleForkDoneHandler(int exitcode, int bysignal) {
  * ## Server hooks implementation
  * -------------------------------------------------------------------------- */
 
+/* This must be synced with REDISMODULE_EVENT_*
```

Comment:
@guybe7 We already have `_REDISMODULE_EVENT_NEXT` defined, perhaps we should add a one-time assertion that `sizeof(moduleEventVersions)` matches the expected size as a precaution?